src/python/grpcio/grpc/_server.py

```diff
@@ -959,6 +959,17 @@ def add_secure_port(self, address, server_credentials):
     def start(self):
         _start(self._state)
 
+    def wait_for_termination(self, grace=None):
```

Comment:
`grace` doesn't appear to be used anywhere.

Reply:
See above.